test/index.js

```diff
@@ -0,0 +1,176 @@
+const expect = require('expect');
+const {createRobot} = require('probot');
+const plugin = require('..');
+const commentEvent = require('./fixtures/issue_comment.created');
+
+describe('snooze', () => {
+  let robot;
+  let github;
+
+  beforeEach(() => {
+    robot = createRobot();
+
+    // Load the plugin
+    plugin(robot);
```

Comment:
> Actually I think this promise is coming from probot itself!

Yes, it's coming from [probot-visitor calling `robot.auth`](https://github.com/probot/visitor/blob/master/index.js#L62). Your test is replacing `robot.auth` below, so if you move this after that, it should allow you to at least get to the next step.